RecoTracker/MkFitCore/src/PropagationMPlex.cc

```diff
@@ -818,17 +818,17 @@ namespace mkfit {
 
     float cosahTmp[NN];
     float sinahTmp[NN];
-    if constexpr (Config::useTrigApprox) {
-#pragma omp simd
+    if (Config::useTrigApprox) {
```

Comment:
You lost the constexpr, probably that's why the internal error went away

Reply:
Doh!